volontulo/frontend/scss/bootstrap/_alerts.scss

```diff
@@ -68,6 +68,6 @@
   @include alert-variant($alert-warning-bg, $alert-warning-border, $alert-warning-text);
 }
 
-.alert-error {
+.alert-danger {
```

Comment:
`.alert-error` is actually custom and 
`.alert-danger` is the original one actually ;}